src/components/AutoComplete.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import debounce from 'lodash.debounce';
+import { peopleFromServer } from '../data/people';
+import { Person } from '../types/Person';
+
+interface AutoCompleteProps {
+  onSelected: (person: Person | null) => void;
+}
+
+export const AutoComplete: React.FC<AutoCompleteProps> = ({ onSelected }) => {
```

Comment:
<img width="610" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/70220667/a5836b2a-87ba-4f9f-8afe-b3aa749596f8">
pass the delay via props